.github/workflows/deploy_doc.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy SDK doc
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  deploy-documentation:
+    name: Deploy technical documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: get last release number
+        run: |
+          last_version_tag=$(curl --silent "https://api.github.com/repos/kili-technology/kili-python-sdk/releases/latest" | jq -r .tag_name)
+          echo $last_version_tag
+          echo "LAST_VERSION=$last_version_tag" >> $GITHUB_ENV
```

Comment:
ah and also I haven't seen this: Here it deploys the doc from the latest release tag.
This is already done automatically in the publish script.
For me, adding a button in Github would only be useful if we can deploy the doc for another branch. For example the use case that Paul just had: I realize that I miss a part of the doc that is on master but not on the last release, then I want to release the doc from master

Reply:
Or maybe a miss something? Why would it be useful to have a button in Github to publish the doc from the last release if it is already automatically released ?

Reply:
Ah I just saw that you raised the issue in your previous comment ;)

Reply:
so the workflow has:
- a workflow_call trigger for when it's called from publish.yml. The doc will be deployed from the release branch (same behavior as before)
- a workflow_dispatch to trigger the workflow from the UI. With the option to choose between checkout to master or checkout to latest release branch (default to master)

Reply:
👌